[2025-04-03T17:20:17.273-0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-03T17:20:17.279-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline.gpSmoker manual__2025-04-03T22:20:06.582106+00:00 [queued]>
[2025-04-03T17:20:17.282-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline.gpSmoker manual__2025-04-03T22:20:06.582106+00:00 [queued]>
[2025-04-03T17:20:17.282-0500] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-03T17:20:17.459-0500] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): gpSmoker> on 2025-04-03 22:20:06.582106+00:00
[2025-04-03T17:20:17.461-0500] {standard_task_runner.py:72} INFO - Started process 1750163 to run task
[2025-04-03T17:20:17.463-0500] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'pipeline', 'gpSmoker', 'manual__2025-04-03T22:20:06.582106+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/exec_pipeline.py', '--cfg-path', '/tmp/tmphl5tahvw']
[2025-04-03T17:20:17.464-0500] {standard_task_runner.py:105} INFO - Job 70: Subtask gpSmoker
[2025-04-03T17:20:17.963-0500] {task_command.py:467} INFO - Running <TaskInstance: pipeline.gpSmoker manual__2025-04-03T22:20:06.582106+00:00 [running]> on host royal-07.cs.wisc.edu
[2025-04-03T17:20:18.373-0500] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='cloud.user' AIRFLOW_CTX_DAG_ID='pipeline' AIRFLOW_CTX_TASK_ID='gpSmoker' AIRFLOW_CTX_EXECUTION_DATE='2025-04-03T22:20:06.582106+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-03T22:20:06.582106+00:00'
[2025-04-03T17:20:18.373-0500] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-03T17:20:18.380-0500] {taskinstance.py:3313} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/airflow/dags/exec_pipeline.py", line 30, in groupby_smoker
    df = pd.read_json(data)
         ^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/pandas/io/json/_json.py", line 791, in read_json
    json_reader = JsonReader(
                  ^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/pandas/io/json/_json.py", line 904, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/pandas/io/json/_json.py", line 944, in _get_data_from_filepath
    self.handles = get_handle(
                   ^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/pandas/io/common.py", line 728, in get_handle
    ioargs = _get_filepath_or_buffer(
             ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/pandas/io/common.py", line 472, in _get_filepath_or_buffer
    raise ValueError(msg)
ValueError: Invalid file path or buffer object type: <class 'airflow.models.xcom.LazyXComSelectSequence'>
[2025-04-03T17:20:18.385-0500] {taskinstance.py:1226} INFO - Marking task as FAILED. dag_id=pipeline, task_id=gpSmoker, run_id=manual__2025-04-03T22:20:06.582106+00:00, execution_date=20250403T222006, start_date=20250403T222017, end_date=20250403T222018
[2025-04-03T17:20:18.757-0500] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-03T17:20:18.757-0500] {standard_task_runner.py:124} ERROR - Failed to execute job 70 for task gpSmoker (Invalid file path or buffer object type: <class 'airflow.models.xcom.LazyXComSelectSequence'>; 1750163)
Traceback (most recent call last):
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 3006, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 274, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 3161, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 3185, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/airflow/dags/exec_pipeline.py", line 30, in groupby_smoker
    df = pd.read_json(data)
         ^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/pandas/io/json/_json.py", line 791, in read_json
    json_reader = JsonReader(
                  ^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/pandas/io/json/_json.py", line 904, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/pandas/io/json/_json.py", line 944, in _get_data_from_filepath
    self.handles = get_handle(
                   ^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/pandas/io/common.py", line 728, in get_handle
    ioargs = _get_filepath_or_buffer(
             ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/udhbhav/miniconda3/lib/python3.11/site-packages/pandas/io/common.py", line 472, in _get_filepath_or_buffer
    raise ValueError(msg)
ValueError: Invalid file path or buffer object type: <class 'airflow.models.xcom.LazyXComSelectSequence'>
[2025-04-03T17:20:18.800-0500] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-04-03T17:20:18.805-0500] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-03T17:20:18.806-0500] {local_task_job_runner.py:245} INFO - ::endgroup::
